lib/classes/record/adapter.php

```diff
@@ -941,7 +941,7 @@ public function getTitle($locale = null, Array $options = [])
             $this->set_data_to_cache(self::CACHE_TITLE, $title);
         }
 
-        return $title;
+        return htmlspecialchars($title);
```

Comment:
NO ! NEVER do encoding at low level. Do it ONLY on the usage, for eg. in the twig.